CefSharp.Example/Resources/Home.html

```diff
@@ -351,6 +351,14 @@ <h3 id="features-javascript-integration">JavaScript integration</h3>
                         objects.
                     </p>
                     <h3>Hooks by which you can modify and/or override certain features of the web browsing</h3>
+                    <p>
+                        A fix to allow params to be used by the .NET methods called by the javascript<br />
```

Comment:
This should be in `BindingTest.html`. Also having ` A fix to allow params to be used by the .NET methods called by the javascript` is not really relevant to someone using `CefSharp`. Simple description of what your demoing would be better :+1: